pallets/amm/src/weights.rs

```diff
@@ -35,58 +35,56 @@
 // --template=./.maintain/frame-weight-template.hbs
 // --output=./pallets/amm/src/weights.rs
 
+
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 #![allow(clippy::all)]
 
-use frame_support::{
-    traits::Get,
-    weights::{constants::RocksDbWeight, Weight},
-};
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
 use sp_std::marker::PhantomData;
 
 /// Weight functions needed for pallet_amm.
 pub trait WeightInfo {
-    fn add_liquidity_non_existing_pool() -> Weight;
-    fn add_liquidity_existing_pool() -> Weight;
-    fn remove_liquidity() -> Weight;
+	fn add_liquidity_non_existing_pool() -> Weight;
+	fn add_liquidity_existing_pool() -> Weight;
+	fn remove_liquidity() -> Weight;
 }
 
 /// Weights for pallet_amm using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-    fn add_liquidity_non_existing_pool() -> Weight {
-        (98_000_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(6 as Weight))
-            .saturating_add(T::DbWeight::get().writes(7 as Weight))
-    }
-    fn add_liquidity_existing_pool() -> Weight {
-        (84_000_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(6 as Weight))
-            .saturating_add(T::DbWeight::get().writes(6 as Weight))
-    }
-    fn remove_liquidity() -> Weight {
-        (83_000_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(7 as Weight))
-            .saturating_add(T::DbWeight::get().writes(6 as Weight))
```

Comment:
maybe we should define `rustfmt.toml`

Reply:
I am running `cargo fmt` before pushing the code.